packages/react-frontend/src/SearchBar.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import "./SearchBar.css";
+import { fetchQuestionsByTitle } from "./apis";
+
+export const SearchBar = ({ setQuestions }) => {
+    const [input, setInput] = useState("");
+    const fetchData = (value) => {
+        fetchQuestionsByTitle(value)
```

Comment:
Cool to see we're using the API with a filter instead of filtering on the front end.

Reply:
Looks good to me